system/settings/settings.xml

```diff
@@ -1075,6 +1075,19 @@
           </dependencies>
           <control type="toggle" />
         </setting>
+	<setting id="filelists.deleteconfirm" type="boolean" label="40801" help="40802">
```

Comment:
I'm not sure this setting belongs here. Is this handling always confirmation when deleting physical files?